### PR TITLE
gsi changes to run against 4.2.8

### PR DESCRIPTION
**Pull request by jassigill2000**

### CODE REVIEW

**Comment by jassigill2000** [comment]
Terrell can you review this.

File: irods_consortium_continuous_integration_build_hook.py [hunk 1]
```diff
@@ -9,65 +9,49 @@
...
+        externals = []
+        for irods_externals in externals_list:
+            externals.append(glob.glob(os.path.join(os_specific_directory, irods_externals + '*.{0}'.format(package_suffix)))[0])
+        irods_python_ci_utilities.install_os_packages_from_files(externals)
+    add_cmake_to_front_of_path()
+    install_os_specific_dependencies()
+
+def install_os_specific_dependencies_apt():
+    irods_python_ci_utilities.install_os_packages(['globus-proxy-utils', 'libglobus-gssapi-gsi-dev', 'make', 'libssl-dev', 'libkrb5-dev', 'gcc'])
+    if irods_python_ci_utilities.get_distribution_version_major() == '16':
```
Comment (trel) [inline irods_consortium_continuous_integration_build_hook.py:41]: so this works fine on ub18?

Comment (jassigill2000) [reply irods_consortium_continuous_integration_build_hook.py]: It builds on ub18, but tests don't pass on ubuntu 18. The tests pass on ubuntu 16 and centOS 7. it fails during setup when the command ```grid-cert-request -nopw -force -cn gsi_client_user``` is run. 
```
stderr: Can't load /var/adm/wtmp into RNG
140420625727936:error:2406F079:random number generator:RAND_load_file:Cannot open file:../crypto/rand/randfile.c:88:Filename=/var/adm/wtmp
Can't load /var/log/messages into RNG
140420625727936:error:2406F079:random number generator:RAND_load_file:Cannot open file:../crypto/rand/randfile.c:88:Filename=/var/log/messages
Error number 1 was returned by 
   /usr/bin/openssl
```

**Comment by jassigill2000** [comment]
I have tested this against ubuntu 16, 18 and centOS 7 and tests pass against 4.2.8

**Pull request merged**